server/jetstream_api.go

```diff
@@ -656,6 +656,8 @@ type JSApiMsgGetRequest struct {
 	UpToSeq uint64 `json:"up_to_seq,omitempty"`
 	// Only return messages up to this time.
 	UpToTime *time.Time `json:"up_to_time,omitempty"`
+	// Return messages as of this time.
```

Comment:
We should move this to the batch ones v s multi. @ripienaar will redo some comments etc for go doc in a separate PR.

Also, mybe StartTime to match what we have on consumer config?